specification/trafficmanager/resource-manager/Microsoft.Network/stable/2020-12-01/trafficmanager.json

```diff
@@ -0,0 +1,1649 @@
+              },
+              "value": {
+                "type": "string",
+                "description": "Header value."
+              }
+            },
+            "description": "Custom header name and value."
+          },
+          "description": "List of custom headers."
+        },
+        "expectedStatusCodeRanges": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "min": {
+                "type": "integer",
+                "description": "Min status code."
+              },
+              "max": {
+                "type": "integer",
+                "description": "Max status code."
+              }
+            },
+            "description": "Min and max value of a status code range."
+          },
+          "description": "List of expected status code ranges."
+        }
+      },
+      "description": "Class containing endpoint monitoring settings in a Traffic Manager profile."
+    },
+    "ProfileProperties": {
+      "properties": {
+        "profileStatus": {
+          "type": "string",
+          "description": "The status of the Traffic Manager profile.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "ProfileStatus",
+            "modelAsString": true
+          }
+        },
+        "trafficRoutingMethod": {
+          "type": "string",
+          "description": "The traffic routing method of the Traffic Manager profile.",
+          "enum": [
+            "Performance",
+            "Priority",
+            "Weighted",
+            "Geographic",
+            "MultiValue",
+            "Subnet"
+          ],
+          "x-ms-enum": {
+            "name": "TrafficRoutingMethod",
+            "modelAsString": true
+          }
+        },
+        "dnsConfig": {
+          "$ref": "#/definitions/DnsConfig",
+          "description": "The DNS settings of the Traffic Manager profile."
+        },
+        "monitorConfig": {
+          "$ref": "#/definitions/MonitorConfig",
+          "description": "The endpoint monitoring settings of the Traffic Manager profile."
+        },
+        "endpoints": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Endpoint"
+          },
+          "description": "The list of endpoints in the Traffic Manager profile."
+        },
+        "trafficViewEnrollmentStatus": {
+          "type": "string",
+          "description": "Indicates whether Traffic View is 'Enabled' or 'Disabled' for the Traffic Manager profile. Null, indicates 'Disabled'. Enabling this feature will increase the cost of the Traffic Manage profile.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "TrafficViewEnrollmentStatus",
+            "modelAsString": true
+          }
+        },
+        "allowedEndpointRecordTypes": {
```

Comment:
Is this a fixed set of values? If yes, consider modeling them as an enum and reference it here.

Reply:
Done, and I've reintroduced the new IPv4 / IPv6 values this API version was created for.